docs-source/content/userguide/managing-fmw-domains/soa-suite/WebLogic-Monitoring-Exporter-Setup.md

```diff
@@ -0,0 +1,298 @@
+# WebLogic Monitoring Exporter Setup to Monitor the SOA Domain Instance
+
+You can monitor the SOA instance using Prometheus and Grafana by exporting the metrics from the domain instance using the WebLogic Monitoring Exporter to the Prometheus.
```

Comment:
the SOA instance -> a SOA domain 
WebLogic Monitoring Exporter to the Prometheus. -> WebLogic Monitoring Exporter.